pkgs/top-level/all-packages.nix

```diff
@@ -2795,7 +2795,9 @@ in
 
   netboot = callPackage ../tools/networking/netboot {};
 
-  netcat = callPackage ../tools/networking/netcat { };
+  netcat = netcat-openbsd;
+
+  netcat-gnu = callPackage ../tools/networking/gnu-netcat { };
```

Comment:
Nitpick: the naming is rather inconsistent. I think it would be better to use `netcat-gnu` even for the directory name.